### DISCUSSION TITLE
Welcome to Hacktoberfest-21-Open-First-PR Discussions!

### DISCUSSION THREAD

**Discussion by iamdevvalecha** [iamdevvalecha/Hacktoberfest-21-Open-First-PR#55] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by imaaquibali** [comment]
hii @iamdevvalecha  bhai aapne abhi tk PR merge nhi ke hai is there any problem in it please let me know bhai

**Reply by iamdevvalecha** [reply]
https://github.com/Binary-Cumates/Hacktoberfest-21-Open-First-PR/pull/68 karli bhai merge

**Comment by imaaquibali** [comment]
Thanks bhai

**Comment by gokulrajpuviyarasu** [comment]
Excluded Project

**Comment by beingDeveloper2312** [comment]
#282 Please review my first PR.

**Comment by Chantelsky** [comment]
@iamdevvalecha You will need to add the labels hacktoberfest2021 and hacktoberfest-accepted for the PRs to be counted in this years hacktoberfest.
